spring/spring-boot-3/remote/src/main/java/org/infinispan/spring/starter/remote/actuator/RemoteInfinispanCacheMeterBinder.java

```diff
@@ -70,8 +66,21 @@ protected void bindImplementationSpecificMetrics(MeterRegistry registry) {
             .description("Time elapsed in seconds since the last statistics reset")
             .register(registry);
 
-      averages(registry);
-      nearCacheMetrics(registry);
+      try {
+         averages(registry);
+         nearCacheMetrics(registry);
+      } catch (HotRodClientException exception) {
+         // Statistics will be all 0L when the connected user is not authorized to read the cache statistic
+      }
+   }
+
+   private Long callMetric(Supplier<Long> supplier) {
+      try {
+         return supplier.get();
+      } catch (HotRodClientException exception) {
```

Comment:
We can check for the presence of "ISPN000287" to ensure that we don't suppress non-authorization related failures.

Reply:
I don't think any wrapping is required: all of the client statistics are local

Reply:
Argh I didn't realise that. I guess it's just the call to `#Size()` that's resulting in the Authorization issue then.

Reply:
I would just remove all of the `callMetrics()` calls.